app/src/sharedTest/java/org/oppia/app/help/HelpFragmentTest.kt

```diff
@@ -38,6 +40,30 @@ class HelpFragmentTest {
     FirebaseApp.initializeApp(ApplicationProvider.getApplicationContext())
   }
 
+  private fun createHelpActivityIntent(profileId: Int, isFromExploration: Boolean): Intent {
+    return HelpActivity.createHelpActivityIntent(
+      ApplicationProvider.getApplicationContext(),
+      profileId,
+      isFromExploration
+    )
+  }
+
+  @Test
+  fun testHelpFragment_getIsFromExplorationTrue_checkBackArrowVisible() {
```

Comment:
Rename to `testHelpFragment_parentIsExploration_checkBackArrowVisible `

Reply:
Done